### PR TITLE
Redirect to /login-error on login error

### PR DESCRIPTION
**Pull request by vogre**

### CODE REVIEW

File: passport/README.md [hunk 1]
```diff
@@ -49,3 +49,7 @@ export async function getTodos() {
   return get(`/todos/${user.id}`);
 });
 ```
+
+### Error handling
+
+If a login fails due to some reason (e.g. misconfiguration of the login provider) the user will be redirected to /login-error.
```
Comment (arielshaqed) [inline passport/README.md]: Please consider adding an example of something useful to do there.

File: passport/src/auth-handler.ts [hunk 2]
```diff
@@ -102,8 +102,7 @@ function createOnCallback(domain: string): express.Handler {
   return (req, res, next) => {
     passport.authenticate(`auth0-${domain}`, (err, user, _info, _extra) => {
       if (err) { return next(err); }
-      // TODO: redirecting to /login, a misconfiguration causes a confusing redirect loop
-      if (!user) { return res.redirect('/login'); }
+      if (!user) { return res.redirect('/login-error'); }
```
Comment (arielshaqed) [inline passport/src/auth-handler.ts]: How does this change the behaviour of existing apps, specifically of current templates with auth and any apps built on top of those?

Comment (vogre) [reply passport/src/auth-handler.ts]: I will submit PRs to update the package in all first-party templates.

File: passport/README.md (hunk 1 above)
Comment (vogre) [reply passport/README.md]: Good idea, added.

**Pull request merged**